addon/objects/uml-primitives/fd-uml-transition.js

```
@@ -61,6 +61,70 @@ export let Transition = Connection.define('flexberry.uml.Transition', {
+  ].join(''),
+
+  initialize: function() {
+    NormalizedDescriptionView.prototype.initialize.apply(this, arguments);
+
+    // Prevent paper from handling pointerdown.
+    this.$box.find('textarea').on('mousedown click', function(evt) {
+      evt.stopPropagation();
+    });
+
+    this.$box.find('.description-input').on('blur', function(evt) {
+      this.addSquareBracketsToDescription($(evt.target));
```

ESLint

[error] Do not use global `$` or `jQuery`

(ember/no-global-jquery)